lib/plugins/aws/deployFunction.js

```diff
@@ -63,6 +65,35 @@ class AwsDeployFunction {
     }
   }
 
+  // TODO: TESTS
```

Comment:
For completeness it'll be nice to address those TODO's. Is this PR in temporary unfinished state?

Reply:
I pushed that commit with intention to discuss the approach, it's definitely not in the final state as it's not cleaned up and tested properly